master/buildbot/www/ldapuserinfo.py

```diff
@@ -26,15 +26,19 @@
 
 from urllib.parse import urlparse
 
-import ldap3
-
 from twisted.internet import threads
 
 from buildbot.util import bytes2unicode
 from buildbot.util import flatten
 from buildbot.www import auth
 from buildbot.www import avatar
 
+try:
+    import ldap3
```

Comment:
This change needs to be split off to a separate PR.

Reply:
Ok, I only included it because I noticed `make trial` was failing on this test because just importing this file would cause the test suite to fail.

Reply:
Moved to #5857